server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -321,7 +321,17 @@ abstract class Host(user: String,
 
   def ips = hostIps.toList
 
-  def calculateCpuAmount = command("lscpu", false).get.split('\n').map(_.split(':')).map(a => a(0) -> a(1)).toMap.getOrElse("CPU(s)", "0").trim.toInt
+  def calculateCpuAmount = {
+    if(UtilCommands.isOSX) {
+      val x = command("sysctl hw.physicalcpu", false).get.split(":")(1).trim.toInt
+      info("mac cpu=" + x)
+      x
+    }
+    else {
+      command("lscpu", false).get.split('\n').map(_.split(':')).map(a => a(0) -> a(1)).toMap.getOrElse("CPU(s)", "0").trim.toInt
+    }
+
+  }
```

Comment:
same comment here. why do you have this code in this FB?

Reply:
I think I need to merge master...

Reply:
Fix pushed